CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contributing to AWS ARC ECS
+
+Thank you for considering contributing to AWS ARC ECS! We appreciate your time and effort.
+To ensure a smooth collaboration, please take a moment to review the following guidelines.
+
+## How to Contribute
+
+1. Fork the repository to your own GitHub account.
+2. Clone the repository to your local machine.
+   ```bash
+   git clone https://github.com/<your_organization>/<your_terraform_module>.git
```

Comment:
should we mention the actual repo instead of placeholders ?

Reply:
Nope @vijay-stephen  , someone will fork then he/she can specify its own repo name so we dont know what will be the repo-name by default it will be our repo name only so placeholders are fine @tsaucier-sf